app/lib/cms/sts/assume_role_web_identity.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module CMS
+  module STS
+    class AssumeRoleWebIdentity
+      include Singleton
+
+      class_attribute :region, :role_arn, :role_session_name, :web_identity_token_file
```

Comment:
my suggestion was to set `identity_credentials` as a class_attribute, everything else is already stored in config as far as I can tell. Is there any advantage to keep these as an attribute but store `identity_credentials` in Rails configuration? i.e. the point was to avoid polluting Rails configuration with with custom objects. And set it regardless of whether we use that or not... as explained in @mayorova's comment.

Reply:
Changes applied in e6d084e.
Now the `CMS::STS::AssumeRoleWebIdentity` Singleton gets its data from ENV variables (through Rails.application.config.s3), as we discussed :v: